presto-docs/src/main/sphinx/optimizer/statistics.rst

```diff
@@ -0,0 +1,53 @@
+================
+Table Statistics
+================
+
+Presto supports statistics based optimizations for queries. For a query to take
+advantage of these optimizations, Presto must have statistical information for
+the tables in that query.
+
+Table statistics are provided to the query planner by connectors.  Currently, the
+only connector that supports statistics is the :doc:`/connector/hive`.
+
+Table Layouts
+-------------
+
+Statistics are exposed to the query planner by a table layout. A table layout
+represents a subset of a table's data and contains information about the
+organizational properties of that data (like sort order and bucketing).
+
+The number of table layouts available for a table and the details of those table
+layouts are specific to each connector.  Using the Hive connector as an example:
+
+* Non-partitioned tables have just one table layout representing all data in the
+  table
```

Comment:
Nit: don't wrap a single word. We wrap to improve readability, but this is less readable